data/campaigns/Dead_Water/scenarios/02_Flight.cfg

```diff
@@ -376,7 +376,7 @@ _f, _f, _f, _f, _f, _f, _f
             [else]
                 [message]
                     speaker=Kai Krellis
-                    message= _ "But will everyone follow me on so perilous an adventure? I am still a youth; it may be they will not trust me."
+                    message= _ "But will everyone follow me on so perilous an adventure? I am still a youth. They might not trust me."
```

Comment:
Saying "It may be they will" instead of "They might" seems like valid extra floweriness for a king; I don't think Kai should be concision-ed.

Reply:
> Saying "It may be they will" instead of "They might" seems like valid extra floweriness for a king; I don't think Kai should be concision-ed.

That's fair to say. Can I request at least one more person's opinion on this conversation?

Reply:
ok how about we ask what @beetlenaut what his original intent was?

Reply:
I'm now thinking this is not worth changing in a Proofread Pass, since the original text is technically okay. Will revert this back.

Reply:
I'm not sure how I missed this PR, but I agree that the original phrasing here is better for a king, even if only an inexperienced one. Seems nemaara is okay with it either way.

Regarding CI, `[ci skip]` still works to cancel the CI on commits, but I suppose it's better to only use it on really small changes.

Reply:
I was trying to make Kai Krellis sound more formal than other characters, so giving him longer, more flowery language was intentional. However, I'm not attached to any specific language in DW because ESR edited a lot of the story and dialog before it was mainlined anyway.